app/backend/src/couchers/db.py

```diff
@@ -13,6 +15,13 @@
 logger = logging.getLogger(__name__)
 
 
+def apply_migrations():
+    alembic_cfg = Config("alembic.ini")
+    # alembic screws up logging config by default, this tells it not to screw it up if being run at startup like this
+    alembic_cfg.set_main_option("dont_mess_up_logging", "False")
```

Comment:
Is this a joke? Or does a configuration parameter with that name?

Reply:
I made it up, alembic changes logging setting by default (for nice terminal output), but if we're running it programmatically, don't want it to modify those settings.

Reply:
I still don't understand, gotta check how alambic works tonight.

Reply:
Running without the line doesn't cause my terminal to mess up... I run py.test in an xterm.

Reply:
It just changes the logging output format (and logging levels, I think)